### ISSUE TITLE
Issues with Circuit Court cases at the JPML

### ISSUE THREAD

**Issue by v-anne** [freelawproject/recap#376]
Under 28 U.S.C. § 2112, when people challenge the same rule issued by a federal agency in multiple courts of appeals, the Judicial Panel on Multidistrict Litigation has to hold a lottery to decide which court they will be consolidated in.

I think RECAP has issues right now with processing dockets of cases in this position. They are generally numbered as MCP [Multicircuit petition] No. XXX, with the latest being MCP No. 190.

In the past, I tried using RECAP to save these dockets after opening them in PACER, but I was unsuccessful.

https://www.jpml.uscourts.gov/multicircuit-petitions

**Comment by mlissner** [comment]
Can you share an example of one of these cases? I thin kit's a corner case we haven't dealt with, and I'm not sure how highly we can prioritize it, but I'd be happy to at least get the details put together so we can think about it carefully.

**Comment by v-anne** [comment]
Here is an example case. I tried RECAPing it and it didn't pick it up.
I got a 404 error from RECAP at this link: https://www.courtlistener.com/recap/gov.uscourts.jpml.1695296/
The button shows up on the page but it doesn't work.
<img width="1782" alt="image" src="https://github.com/user-attachments/assets/63c94f6a-ad28-4936-a548-7c3e87d86cf9">
<img width="1782" alt="image" src="https://github.com/user-attachments/assets/1ed6de7f-3ca4-46df-8865-de238f865a7c">

**Comment by mlissner** [comment]
Yeah, that looks different enough that I'm not entirely surprised. I think we'd need to do a fair amount of work to make this work properly. The extension would need to detect it properly, and then we'd probably need to update our docket parsers in Juriscraper, and maybe even our merging code in CL itself. 

Hrm...that's a good deal of work for an edge case like this. If we help guide you through, would you be interested in taking a stab at it?

**Comment by johnhawkinson** [comment]
I think Bill @flooie spent a substantial amount of time thinking about JPML issues recently.
That said, I also think that almost nobody is interested in the JPML dockets.
For MDLs, the thing that is of real interest is the district court docket at the transferee court, and we handle that just fine.
Which is to say I agree this is an issue but I suspect it's not terribly pressing for…almost anyone. Not sure if v-anne is the exception that proves my rule or not or what that idiom even means in this context.

**Comment by flooie** [comment]
@johnhawkinson I definitely looked into these when working on expanding docket numbers. 

We could've added an mdl model field and used it to link things together but I think John is probably right that identifying the eventual lottery winner is what most people would want.

**Comment by v-anne** [comment]
I'm open to working on a feature for this, but if it's truly an edge case, it might not be worth it. To be clear, my interest was more in the circuit court lotteries than the district court MDLs, which might interest even fewer people.

**Comment by mlissner** [comment]
Over in the dup, @johnhawkinson [says](https://github.com/freelawproject/recap/issues/393#issuecomment-2746001977): 

> Perhaps thr extension should try to acknowledge this is not going to work
> (e.g. "RECAP does not support JPML dockets") instead of pretending.

Fair point. Still, JPML hasn't been an issue, really, for over a decade, so I find it hard to prioritize unless something has changed. This would definitely be a welcome change, but we need to stay focused on higher priorities for now. Sorry.
